docs/yaml_docs/pipeline_ingestion_yml.md

```diff
@@ -13,6 +13,17 @@
 In this documentation, the parameters of the `ingest` configuration yaml file are explained. 
 This file is generated running `panpipes ingest config`.  <br> The individual steps run by the pipeline are described in the [spatial preprocess workflow](../workflows/qc.md). 
 
+When running the ingestion workflow, panpipes provides a basic `pipeline.yml` file.
+To run the workflow on your own data, you need to specify the parameters described below in the `pipeline.yml` file to meet the requirements of your data.
+However, we do provide pre-filled versions of the `pipeline.yml` file for individual tutorials.
+You can download the different ingestion pipeline.yml files here:
+- Basic `pipeline.yml` file (not prefilled): [Download here](https://github.com/DendrouLab/panpipes/blob/main/panpipes/panpipes/pipeline_ingest/pipeline.yml)
```

Comment:
Please note, this is the file that is generated when calling `panpipes ingest config`